From 52b4083771e4dc66c05cbc379c1c5b4213076216 Mon Sep 17 00:00:00 2001
From: maivananh111 <aaanh01@gmail.com>
Date: Fri, 1 Aug 2025 23:17:49 +0700
Subject: [PATCH 4/5] gpio: rockchip: Update debounce config function

---
 drivers/gpio/gpio-rockchip.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpio/gpio-rockchip.c b/drivers/gpio/gpio-rockchip.c
index 4a0c12a2808d..14cb0bfc2c95 100644
--- a/drivers/gpio/gpio-rockchip.c
+++ b/drivers/gpio/gpio-rockchip.c
@@ -292,15 +292,15 @@ static int rockchip_gpio_set_config(struct gpio_chip *gc, unsigned int offset,
 				  unsigned long config)
 {
 	enum pin_config_param param = pinconf_to_config_param(config);
+	unsigned int debounce = pinconf_to_config_argument(config);
 
 	switch (param) {
 	case PIN_CONFIG_INPUT_DEBOUNCE:
-		rockchip_gpio_set_debounce(gc, offset, 0);
 		/*
 		 * Since Rockchip's GPIO hardware debounce function does not
 		 * support configuring individual pins, it will not be used.
 		 */
-		return -ENOTSUPP;
+		return rockchip_gpio_set_debounce(gc, offset, debounce);
 	default:
 		return -ENOTSUPP;
 	}
-- 
2.34.1

